src/RemoteSigner.js

```diff
@@ -80,23 +80,31 @@ class RemoteSigner extends Ethers.Signer {
 
   /**
    * Sign a hashed piece of data
-   * @param {String} digest - Hex string of hashed data
+   * @param {String} digest - string of hashed data
+   * @param {Boolean} usePersonal - use EIP-191 personal_sign
```

Comment:
please change to `@param {boolean=} usePersonal=false` to indicate it is optional and defaults to false